glide.yaml

```diff
@@ -1,23 +1,25 @@
 package: github.com/mainflux/mainflux
 license: Apache License 2.0
 import:
-  - package: github.com/cisco/senml
-  - package: github.com/dgrijalva/jwt-go
-    version: 3.0.0
-  - package: github.com/gocql/gocql
-  - package: github.com/go-kit/kit
-    version: 0.4.0
-  - package: github.com/go-zoo/bone
-    version: 1.2
-  - package: github.com/nats-io/go-nats
-    version: 1.3.0
-  - package: github.com/prometheus/client_golang
-    version: 0.8.0
-  - package: github.com/sirupsen/logrus
-    version: 1.0.3
-  - package: golang.org/x/crypto
-    subpackages:
-      - bcrypt
+- package: github.com/cisco/senml
+- package: github.com/dgrijalva/jwt-go
+  version: 3.0.0
+- package: github.com/gocql/gocql
+- package: github.com/go-kit/kit
+  version: 0.4.0
+- package: github.com/go-zoo/bone
+  version: "1.2"
+- package: github.com/nats-io/go-nats
+  version: 1.3.0
+- package: github.com/prometheus/client_golang
+  version: 0.8.0
+- package: github.com/sirupsen/logrus
+  version: 1.0.3
+- package: golang.org/x/crypto
+  subpackages:
+  - bcrypt
+- package: go.uber.org/zap
```

Comment:
I'd rather that we replace logrus completely instead of adding another logger to the list of dependencies.